llvm/lib/Target/AArch64/AArch64AsmPrinter.cpp

```diff
@@ -104,6 +104,8 @@ class AArch64AsmPrinter : public AsmPrinter {
 
   void LowerJumpTableDest(MCStreamer &OutStreamer, const MachineInstr &MI);
 
+  void LowerHardenedBRJumpTable(const MachineInstr &MI);
```

Comment:
Nit: I think that it's OK not to pass `MCStreamer &OutStreamer` as it's done in other similar member functions (we have `OutStreamer` as a class member so no need to manually pass that), but, if we want to stick with conventions already present here, you might probably want to consume `OutStreamer` as a function argument. This convention might probably be changed in a follow-up refactoring patch - it looks like that there is no reason why we pass `OutStreamer` as an argument now.

Feel free to ignore.

Reply:
I think it's only worth adding this when it's actually needed.